client/components/account-balances/test/index.test.tsx

```diff
@@ -64,3 +188,116 @@ describe( 'AccountBalancesHeader', () => {
 		getByText( expectedGreeting );
 	} );
 } );
+
+describe( 'AccountBalancesTabPanel', () => {
+	test( 'renders the correct tab title and currency data', () => {
+		mockGetCurrencyTabTitle.mockReturnValue( 'USD Balance' );
+		mockOverviews( [ createMockOverview( 'usd', 10000, 20000 ) ] );
+
+		render( <AccountBalancesTabPanel /> );
+
+		const availableAmount = document.querySelector(
+			'.wcpay-account-available__balance_item .wcpay-account-balances__balances__item__amount'
+		) as HTMLElement;
+
+		const pendingAmount = document.querySelector(
+			'.wcpay-account-pending__balance_item .wcpay-account-balances__balances__item__amount'
+		) as HTMLElement;
+
+		// Check the available and pending amounts are rendered correctly.
+		expect( availableAmount ).toHaveTextContent( '$200.00' );
+		expect( pendingAmount ).toHaveTextContent( '$100.00' );
+	} );
```

Comment:
When writing React tests with `react-testing-library`, we're encouraged to avoid using specific implementation details, e.g. className. [see RTL intro docs](https://testing-library.com/docs/react-testing-library/intro)/

My suggestion below would allow us to refactor some implementation details of the component (e.g. change the className) and the test would still be accurate.

**Note:** I added a `title` attribute to the `p` element that renders the balance amount in `<BalanceBlock />`.

```suggestion
test( 'renders the correct tab title and currency data', () => {
		mockGetCurrencyTabTitle.mockReturnValue( 'USD Balance' );
		mockOverviews( [ createMockOverview( 'usd', 10000, 20000 ) ] );

		// Use a query method returned by the render function: (you could also use `container` which will represent `document`)
		const { getByText, getByTitle } = render( <AccountBalancesTabPanel /> );

		// Check the label text. If these don't exist, it will fail, no need to `expect()`
		getByText( 'Available funds' );
		getByText( 'Pending funds' );

		const availableAmount = getByTitle( 'Available funds' );
		const pendingAmount = getByTitle( 'Pending funds' );

		// Check the available and pending amounts are rendered correctly.
		expect( availableAmount ).toHaveTextContent( '$200.00' );
		expect( pendingAmount ).toHaveTextContent( '$100.00' );
	} );
```

Reply:
This is much nicer 😍 

I didn't realise you could query for elements like that. The https://testing-library.com/docs/react-testing-library/intro/ is a good resource to have in my back pocket.